src/main/java/io/vertx/json/schema/OutputUnit.java

```diff
@@ -149,6 +153,74 @@ public OutputUnit addAnnotations(List<OutputUnit> annotations) {
     return this;
   }
 
+  @GenIgnore
+  public OutputUnit setSchemaLocation(String schemaLocation) {
+    this.schemaLocation = schemaLocation;
+    return this;
+  }
+
+  public void checkValidity() throws JsonSchemaValidationException {
```

Comment:
I've tested `checkValidity` in vertx-openapi and it works as expected, but a test in vertx-json-schema to detect regressions could be beneficial.